### ISSUE TITLE
v1.1.0 doesn't work anymore

### ISSUE THREAD

**Issue by netflash** [nosborn/github-action-markdown-cli#5]
Hi there,

Just tried to add it and this is what I got

```
2137f333b9e3: Pull complete
Digest: sha256:ae1822c17b0087cb1eea794e5a293d56cc1fe01f01ef5494d0687c1ef9584239
Status: Downloaded newer image for node:lts-alpine
 ---> 69c8cc9212ec
Step 2/4 : RUN npm install --global --production markdownlint-cli@0.18   && apk --update --no-cache add curl=~7.64 jq=~1.6
 ---> Running in 18d84ca8d6c0
/usr/local/bin/markdownlint -> /usr/local/lib/node_modules/markdownlint-cli/markdownlint.js
+ markdownlint-cli@0.18.0
added 33 packages from 26 contributors in 1.693s
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  curl-7.66.0-r0:
    breaks: world[curl~7.64]
The command '/bin/sh -c npm install --global --production markdownlint-cli@0.18   && apk --update --no-cache add curl=~7.64 jq=~1.6' returned a non-zero code: 1
```

**Comment by nosborn** [comment]
Hi @netflash, thanks for the report. I've removed the over-zealous package pinning and released v1.1.1 to fix.

**Comment by netflash** [comment]
Thank you!
